docs/source/additional/release.rst

```diff
@@ -31,6 +31,21 @@ This is an overview of the main changes, please have a look at the changelog of
 - `backend changelog <https://github.com/Substra/substra-backend/blob/main/CHANGELOG.md>`__
 - `orchestrator changelog <https://github.com/Substra/orchestrator/blob/main/CHANGELOG.md>`__
 
+Substra 0.33.0 --- 2023-10-06
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Substra:**
+
+- **BREAKING**: Substra tools now only provides a unique ``latest`` docker image for each supported Python version.
+
+**Orchestrator and Backend:**
+
+- A new Skaffold profile ``three-orgs`` have been added in the Orchestrator and the Substra-backend. It allows local deployment of a third organization and must be activated on the two repositories. To use it:
+
+  .. code-block:: bash
+
+    skaffold run -p three-orgs
```

Comment:
Is this relevant to users?

Reply:
I thought it was only meant for the doc examples in the CI.

I'm not sure this message deserves to be here but IMO if it does it should also be mentioned in [the guide](https://github.com/Substra/substra-documentation/blob/release-0.33.0/docs/source/how-to/developing-substra/local-deployment.rst) and this paragraph should make it clear it's not a new backend/orc feature but a new local dev feature.

Reply:
The CI was the need, but now that it's here I don't see why not using it. Second point is fair. Updated. WDTY?

Reply:
👍 ❗